### PR TITLE
A number of updates

### PR DESCRIPTION
**Pull request by kevmoo**
Happy to do a clean ff-commit if you LGTM

### CODE REVIEW

File: lib/src/package_graph/package_graph.dart [hunk 1]
```diff
@@ -19,9 +19,7 @@ class PackageGraph {
 
   /// Creates a [PackageGraph] given the [root] [PackageNode].
   factory PackageGraph.fromRoot(PackageNode root) {
-    final allPackages = <String, PackageNode>{
-      root.name: root,
-    };
+    final allPackages = <String, PackageNode>{root.name: root,};
```
Comment (jakemac53) [inline lib/src/package_graph/package_graph.dart]: lets remove this trailing comma in the map literal now that its inlined

File: test/transformer/transformer_test.dart [hunk 2]
```diff
@@ -75,34 +75,43 @@ main() {
...
-    'a|web/a.txt.copy': 'hello',
-  }, {}, messages: [
-    _fileExistsError("CopyBuilder", ["a|web/a.txt.copy"]),
-  ], expectBarbackErrors: true);
+  testPhases(
+      'cant overwrite files',
+      [
+        [singleCopyTransformer]
+      ],
+      {'a|web/a.txt': 'hello', 'a|web/a.txt.copy': 'hello',},
```
Comment (jakemac53) [inline test/transformer/transformer_test.dart]: If you don't mind removing the trailing commas in these map literals as well that would be cool ;)

**Comment by jakemac53** [comment]
lgtm, feel free to ignore the trailing comma nits if you want

File: test/transformer/transformer_test.dart (hunk 2 above)
Comment (kevmoo) [inline test/transformer/transformer_test.dart]: done

File: lib/src/package_graph/package_graph.dart (hunk 1 above)
Comment (kevmoo) [inline lib/src/package_graph/package_graph.dart]: done

**Pull request merged**